src/nvme/ioctl.h

```diff
@@ -4044,7 +4052,7 @@ struct nvme_copy_args {
 	int fd;
 	__u32 timeout;
 	__u32 nsid;
-	__u32 ilbrt;
+	__u64 ilbrt;
```

Comment:
This breaks the stable API.

Reply:
Made changes to set the ilbrt field accordingly based on the version of the structure.